test/clib/test_clib.cpp

```diff
@@ -76,6 +76,12 @@ TEST(ct, soln_objects)
 
     int trans = soln_kinetics(ref2);
     ASSERT_EQ(trans, 1);
+    int buflen = trans_transportModel(trans, 0, 0);
+    char* buf = new char[buflen];
+    trans_transportModel(trans, buflen, buf);
+    string trName = buf;
+    ASSERT_EQ(trName, "mixture-averaged");
+    delete[] buf;
```

Comment:
If you want to avoid needing to do a manual new/delete, you can write
```suggestion
    vector<char> buf(buflen);
    trans_transportModel(trans, buflen, buf.data());
    string trName(buf.data());
    ASSERT_EQ(trName, "mixture-averaged");
```